manager-bundle/skeleton/config/config.yaml

```diff
@@ -52,6 +52,7 @@ framework:
             contao_prio_low: doctrine://default?table_name=tl_message_queue&queue_name=prio_low&auto_setup=false
         routing:
             'Symfony\Component\Mailer\Messenger\SendEmailMessage': contao_prio_high
+            # TODO: Remove these interfaces and their routing configuration in Contao 6
```

Comment:
What "routing configuration" exactly?

Reply:
The ones below this comment, as the interfaces are now deprecated and will be removed in Contao 6.

Reply:
Ah, "routing" means "assignment to a queue". So basically just "remove the following interfaces in Contao 6" 😄

Reply:
Yeah, routing is the term Symfony uses. Hence the config name ;)